src/tests/ftest/dfuse/pil4dfs_dcache.py

```diff
@@ -0,0 +1,329 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import re
+
+from ClusterShell.NodeSet import NodeSet
+from dfuse_test_base import DfuseTestBase
+from dfuse_utils import Pil4dfsDcacheCmd
+
+
+class Pil4dfsDcache(DfuseTestBase):
+    """Test class Description: Runs set of unit test on pil4dfs directory cache.
+
+    :avocado: recursive
+    """
+
+    __tests_suite__ = {
```

Comment:
Unless I'm mistaken in python a `_` prefix means something is "private", a `__` prefix means something is actually private and `__` prefix with `__` suffix is reserved for python internals.  Unless this is overriding some avocado default then I'd expect this to have a single `_` prefix.

Reply:
I was thinking that python naming convention was used to respectively identify protected and private member with `_` and `__`.  However, it seems that I was wrong https://realpython.com/python-double-underscore/#public-interfaces-and-naming-conventions-in-python

Thus, I will use a `_` as you suggested.

- [ ] Use _ instead of __

Reply:
> I was thinking that python naming convention was used to respectively identify protected and private member with `_` and `__`. However, it seems that I was wrong https://realpython.com/python-double-underscore/#public-interfaces-and-naming-conventions-in-python
> 
> Thus, I will use a `_` as you suggested.
> 
> * [x]  Use _ instead of __

Fixed with commit 38b3dbc77